composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
It looks like an upgrade to font-awesome 4.4 and compatibility with SF 3 is the bulk of it though.
Sounds like just something we want as we're trying to make SSA SF 3 compatible, right?